### PR TITLE
Spurs TV 2.9.0

### PR DESCRIPTION
**Pull request by LS80**

### CODE REVIEW

File: plugin.video.spurs-tv/resources/language/English/strings.xml [hunk 1]
```diff
@@ -9,7 +9,9 @@
   <string id="30014">All</string>
```
Comment (learningit) [inline plugin.video.spurs-tv/resources/language/English/strings.xml]: Next update please use "strings.po" in plaxce of of "strings.xml", which is deprecated.

**Comment by learningit** [comment]
The version of the source in https://github.com/LS80/plugin.video.spurs-tv.git is 2.8.0 and I can't check a few things, please update accordingly.

File: plugin.video.spurs-tv/changelog.txt [hunk 2]
```diff
@@ -1,87 +1,90 @@
...
+ - Fixed Tour 2016 category
+
+v2.8.0 (2016-03-19)
  - Fixed YouTube playlists
  - Updated fanart
 
-2.7.0
+v2.7.0 (2016-02-28)
  - Kodi Krypton is now a requirement
  - Use Rollbar for automatically reporting errors
```
Comment (learningit) [inline plugin.video.spurs-tv/changelog.txt]: You may want to make the Rollbar reporting controllable by a user option - some user's may not want this.

**Comment by LS80** [comment]
I'll create a new version with some changes and open a new PR

**Pull request closed without merging**